app/data/menuitems.js

```diff
@@ -32,12 +32,12 @@ export default [
       {
         id: 6.1,
         url: 'http://2017.jsheroes.io/',
-        label: 'JSHeroes 2017',
+        label: '2017',
       },
       {
         id: 6.2,
         url: '/2018',
```

Comment:
can you use next/link for relative urls to avoid full page reload?

Reply:
I had a key here that says internal?

Reply:
Done :)